setup.py

```diff
@@ -3,10 +3,10 @@
 setup(
     name="ray_lightning",
     packages=find_packages(where=".", include="ray_lightning*"),
-    version="0.3.0",
+    version="0.4.0",
     author="Ray Team",
     description="Ray distributed strategies for Pytorch Lightning.",
     long_description="Custom Pytorch Lightning distributed strategies "
     "built on top of distributed computing framework Ray.",
     url="https://github.com/ray-project/ray_lightning_accelerators",
-    install_requires=["pytorch-lightning==1.6.*", "ray"])
+    install_requires=["pytorch_lightning>=1.6.4,<=1.7.1", "ray"])
```

Comment:
is 1.7.0 and 1.7.1 supported in master currently?

Reply:
cc @woshiyyya